### ISSUE TITLE
prevent/ignore ONLY join messages

### ISSUE THREAD

**Issue by 2b2tplayer** [zeroeightysix/KAMI#35]
just a way to detect join messages and somehow erase them from the chat without ignoring the person who has them on

**Comment by zeroeightysix** [comment]
What about a module to just ignore chat messages that contain keywords you specify? (or match a regex)

**Comment by VADemon** [comment]
Selective chat based on regex would be awesome.

**Comment by wine** [comment]
You can just cancel the chat event if the message is from the server and contains "joined the game"

**Comment by zeroeightysix** [comment]
@wine Yes but that's so restrictive! Let's aim for extensive functionality in KAMI, shall we?

**Comment by wine** [comment]
@zeroeightysix roger that
